CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Fixed selection of words on right-click in the editor
 - Editor word selection on right-click now accepts both `'` and `’`
 - Add Fortran syntax highlighting (keyword `fortran`)
+- Zettlr can ask you now where you want to store the exported PDF
```

Comment:
There's a merge conflict on the Changelog, but that is to be expected, we can simply fix that one as the very last step!